Add note about Telemetry dashboards for Grafana (#751)
akka/akka-platform-guide: docs-source/docs/modules/deployment/partials/connect-to-grafana.adoc

kubectl port-forward --namespace default service/grafana 3000:80
----

> You can then open your browser and access the Grafana UI at <http://localhost:3000> and login with the user and password that you got above. If you navigate to <http://localhost:3000/datasources> you will see the "Cinnamon Prometheus" datasource that was created automatically when running `pulumi up`. And if you navigate to <http://localhost:3000/dashboards> you will see the https://developer.lightbend.com/docs/telemetry/current/visualizations/grafana.html[Grafana dashboards from Lightbend Telemetry] were installed too.

Reviewer: ```suggestion
You can then open your browser and access the Grafana UI at <http://localhost:3000>. log in with the user and password that you got above. If you navigate to <http://localhost:3000/datasources> you will see the "Cinnamon Prometheus" data source that was created automatically when running `pulumi up`. If you navigate to <http://localhost:3000/dashboards> you will see the https://developer.lightbend.com/docs/telemetry/current/visualizations/grafana.html[Grafana dashboards from Lightbend Telemetry] were also installed.
```
